[OSDOCS-18277] NFV ShiftStack tweaks for RHOSO18 (#111916)
openshift/openshift-docs: modules/installation-openstack-sr-iov-requirements.adoc

:_mod-docs-content-type: REFERENCE
[id="installation-openstack-sr-iov-requirements_{context}"]
= Requirements for clusters on {rh-openstack} or {rhoso} that use SR-IOV

[role="_abstract"]
To use single-root I/O virtualization (SR-IOV) with your deployment, you must meet the following requirements:

// TODO: Verify RHOSP 17.1 is still a supported platform for this OCP version.

> * If you use {rh-openstack-first}, link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_network_functions_virtualization/plan-sriov-deploy_rhosp-nfv[plan your {rh-openstack} SR-IOV deployment].

Reviewer: This is directing to 17.1 docs

* If you use {rhoso-first}, link:https://docs.redhat.com/en/documentation/red_hat_openstack_services_on_openshift/18.0/html/deploying_a_network_functions_virtualization_environment/plan-sriov-deploy_rhoso-nfv[plan your {rhoso} SR-IOV deployment] by referring to _Deploying a network functions virtualization environment_.

* {product-title} must support the NICs that you use. For a list of supported NICs, see "About Single Root I/O Virtualization (SR-IOV) hardware networks" in the "Hardware networks" subsection of the "Networking" documentation.

* For each node that will have an attached SR-IOV NIC, your OpenStack cluster must have:

   ** One instance from the quota
   ** One port attached to the machines subnet
   ** One port for each SR-IOV Virtual Function
   ** A flavor with at least 16 GB memory, 4 vCPUs, and 25 GB storage space

* SR-IOV deployments often employ performance optimizations, such as dedicated or isolated CPUs. For maximum performance, configure your underlying OpenStack deployment to use these optimizations, and then run {product-title} compute machines on the optimized infrastructure.

> // TODO: Verify RHOSP 17.1 is still a supported platform for this OCP version.

Reviewer: RHOSP17.1 is not supported with OCP 4.22

> ** If you use {rh-openstack}, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_the_compute_service_for_instance_creation/configuring-compute-nodes-for-performance#configuring-compute-nodes-for-performance[Configuring Compute nodes for performance].

Reviewer: link is broken 404 not found

> ** If you use {rh-openstack}, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_the_compute_service_for_instance_creation/configuring-compute-nodes-for-performance#configuring-compute-nodes-for-performance[Configuring Compute nodes for performance].

Author: ```suggestion
** If you use {rh-openstack}, see link:https://docs.redhat.com/en/documentation/red_hat_openstack_platform/17.1/html/configuring_the_compute_service_for_instance_creation/assembly_configuring-cpus-on-compute-nodes[Configuring CPUs on Compute nodes].
```
